### PR TITLE
[DEV-2506] add limitRequest in supportIssue

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/subdomains/supporting/support-issue/services/support-issue.service.ts [hunk 1]
```diff
@@ -29,8 +31,23 @@ export class SupportIssueService {
...
     private readonly messageRepo: SupportMessageRepository,
     private readonly supportIssueNotificationService: SupportIssueNotificationService,
+    private readonly limitRequestService: LimitRequestService,
   ) {}
 
+  async createIssueInternal(userData: UserData, dto: CreateSupportIssueInternalDto): Promise<void> {
+    const newIssue = this.supportIssueRepo.create({ userData, ...dto });
+
+    const existingIssue = await this.supportIssueRepo.findOneBy({
+      userData: { id: userData.id },
```
Comment (davidleomay) [inline src/subdomains/supporting/support-issue/services/support-issue.service.ts:41]: `.existsBy`

File: src/subdomains/supporting/support-issue/services/limit-request-job.service.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    const entities = await this.limitRequestRepo.find({
+      where: { supportIssue: IsNull() },
+      relations: { userData: true },
+    });
+
+    for (const entity of entities) {
+      await this.supportIssueService.createIssueInternal(entity.userData, {
+        name: entity.userData.firstname ?? '-',
+        type: SupportIssueType.LIMIT_REQUEST,
```
Comment (davidleomay) [inline src/subdomains/supporting/support-issue/services/limit-request-job.service.ts:33]: `completeName`?

File: src/subdomains/supporting/support-issue/services/support-issue.service.ts [hunk 3]
```diff
@@ -52,6 +69,9 @@ export class SupportIssueService {
       newIssue.additionalInformation = dto.transaction;
     }
 
+    // limit request
+    if (dto.limitRequest) await this.limitRequestService.increaseLimitInternal(dto.limitRequest, userData);
```
Comment (davidleomay) [inline src/subdomains/supporting/support-issue/services/support-issue.service.ts:73]: Create the limit request with support issue? Or is this updated when removing the limit request API?

File: src/subdomains/supporting/support-issue/services/limit-request-job.service.ts (hunk 2 above)
Comment (Yannick1712) [reply src/subdomains/supporting/support-issue/services/limit-request-job.service.ts:33]: We always use the "du"/firstname so thats the reason I used the firstname and not the completeName. But we can change it here if you want to 👍

File: src/subdomains/supporting/support-issue/services/support-issue.service.ts (hunk 3 above)
Comment (Yannick1712) [reply src/subdomains/supporting/support-issue/services/support-issue.service.ts:73]: Yes right, we create a new limitRequest when creating a limitRequestSupportIssue. I think I don't understand your question 😅 
Why should it be updated when removing the limitRequest Endpoints? The limitRequest entity/service will still be used when the endpoints are removed

File: src/subdomains/supporting/support-issue/services/limit-request-job.service.ts (hunk 2 above)
Comment (davidleomay) [reply src/subdomains/supporting/support-issue/services/limit-request-job.service.ts:33]: Ok, I just thought that most users will probably enter their full name when creating a support issue.

File: src/subdomains/supporting/support-issue/services/limit-request.service.ts [hunk 4]
```diff
@@ -25,31 +25,36 @@ export class LimitRequestService {
...
         `${Util.isoDateTime(new Date())}_limit-request_user-upload_${dto.documentProofName}`,
         buffer,
         contentType as ContentType,
       );
+
+      return void this.increaseLimitInternal({ ...dto, documentProofUrl }, user);
     }
 
+    return void this.increaseLimitInternal(dto, user);
+  }
```
Comment (davidleomay) [inline src/subdomains/supporting/support-issue/services/limit-request.service.ts:50]: I think this doesn't wait 🤔  just use `await this.increaseLimitInternal(dto, user);`

**Review by davidleomay — APPROVED** [review]

**Pull request merged**